sdccc/src/main/java/com/draeger/medical/sdccc/manipulation/Manipulations.java

```diff
@@ -135,6 +135,15 @@ ResponseTypes.Result setSystemSignalActivation(
      */
     ResponseTypes.Result triggerDescriptorUpdate(String handle);
 
+    /**
+     * Trigger a descriptor update for some descriptor handle.
```

Comment:
There is no mention of a handle in the method signature, so I would suggest keeping the doc short and simple and only keep the first sentence. Also, not the handle but the descriptor would be updated.

Reply:
Corrected the documentation.